client/my-sites/checkout/checkout-thank-you/akismet-checkout-thank-you.tsx

```diff
@@ -61,6 +71,15 @@ const AkismetCheckoutThankYou: FunctionComponent< AkismetCheckoutThankYouProps >
 						{ strong: <strong /> }
 					) }
 				</p>
+
+				<Button
+					primary
+					busy={ isLoading }
```

Comment:
This button doesn't depend on anything to be in a ready state. However, since we are going to show the API key the user has just purchased here, I've decided to have the button in a busy state while `isLoading` is `true`.